Lib/pty.py

```diff
@@ -66,26 +47,6 @@ def _open_terminal():
             return (fd, '/dev/tty' + x + y)
     raise OSError('out of pty devices')
 
-def slave_open(tty_name):
```

Comment:
The function is still used in the fallback path in openpty() (line 35 above)

Reply:
Oh, thanks a lot! I've missed it.